test/storage-luatest/storage_1_1_test.lua

```diff
@@ -261,3 +261,87 @@ test_group.test_on_bucket_event = function(g)
         box.space.data2:drop()
     end)
 end
+
```

Comment:
Please, fix checkpatch warning in commit msg

Reply:
Done.